file_upload_handler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "file_upload_handler",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Should add a command here to start the server
e.g
```
"start": "node ./server.js"
```
Then you can run just run `npm start` to start the server

Reply:
You've added the command, but it isn't going to actually work with how things are set up atm...